src/components/themes.cy.tsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import styled, { DefaultTheme, rainbowPalette, standardPaletteList } from "styled-components";
+import StyleManager, { lightTheme, darkTheme } from "@/app/themes";
+import { Result } from "axe-core";
+
+const checkColorContrast = (): void => {
+    const terminalLog = (violations: Result[]): void => {
+        cy.task(
+            "table",
+            violations.map(({ id, impact, description, nodes }) => ({
+                id,
+                impact,
+                description,
+                length: nodes.length,
+            }))
+        );
+    };
+
+    cy.injectAxe();
+    cy.checkA11y(undefined, { runOnly: { type: "tag", values: ["wcag2aa"] } }, terminalLog);
+};
+const ForegroundWithBackground: React.FC<{
+    background: string;
+    foreground: string;
+    largeForeground: string;
+}> = (props) => {
+    const StyledH1 = styled.h1`
```

Comment:
Should these styled components be outside of this FC?

Reply:
Oops this is VFB-16 
Let me fix in that branch instead

Reply:
It can't be moved outside as the styledComponent needs the props from the function to change the color and background color